radix-engine/src/kernel/kernel.rs

```diff
@@ -37,6 +37,17 @@ pub struct KernelBoot<'g, V: SystemCallbackObject, S: SubstateStore> {
 }
 
 impl<'g, 'h, V: SystemCallbackObject, S: SubstateStore> KernelBoot<'g, V, S> {
+    pub fn start_test_kernel(&mut self) -> Kernel<SystemConfig<V>, S> {
```

Comment:
Can we add a comment on what this is for, since this should only be used by testing